### DISCUSSION TITLE
Possible to turn off filler word removal to 'detect' them instead?

### DISCUSSION THREAD

**Discussion by coderbec** [openai/whisper#896] [Ideas]
The idea I have is to use it as a presentation trainer. If I could detect the prevalence of filler words like "um, like" etc and report on it - or even produce an alert in real-time of some type, it could be used for presentation training.

**Comment by glangford** [comment]
See
* https://github.com/openai/whisper/discussions/855

and the other discussions referred to there.

**Comment by LaurinmyReha** [comment]
checkout this model which was specifically designed with filler detection in mind:
https://huggingface.co/nyrahealth/CrisperWhisper

and the accompanying repo and paper:
https://github.com/nyrahealth/CrisperWhisper
